src/@next/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Button, PrimaryButton } from '../Button';
+import { Neutral } from '../utilities/colors';
+import {
+  EmptyStateContainer,
+  EmptyStateContentContainer,
+  StyledButtonGroup,
+  StyledDescription,
+  StyledHelpText,
+  StyledImage,
+  StyledTitle,
+} from './EmptyStateStyle';
+
+export enum ImageName {
+  EMPTY_CARTON = 'empty-carton',
+  EMPTY_MAILBOX = 'empty-mailbox',
+  SAFETY_CONE = 'safety-cone',
+}
+
+export interface EmptyStateProps {
+  buttonBasicContent?: React.ReactNode;
+  buttonPrimaryContent?: React.ReactNode;
+  description?: string;
+  fullWidth?: boolean;
+  helpText?: string;
+  imageName?: ImageName;
+  title?: string;
+}
+
+export const EmptyState = ({
+  buttonBasicContent,
+  buttonPrimaryContent,
+  description,
+  fullWidth = true,
+  helpText,
+  imageName,
+  title,
+}: EmptyStateProps) => {
+  const showPrimaryButton = !!buttonPrimaryContent;
+  const showBasicButton = !!buttonBasicContent;
+  const displayButtons = showPrimaryButton || showBasicButton;
+  return (
+    <EmptyStateContainer>
+      {imageName && (
+        <StyledImage src={require(`../../../assets/image/${imageName}.png`)} />
+      )}
+      <EmptyStateContentContainer data-full-width={fullWidth}>
+        {title && (
+          <StyledTitle variant="subtitle1" color={Neutral.B18}>
+            {title}
+          </StyledTitle>
+        )}
+        {description && (
+          <StyledDescription variant="body1" color={Neutral.B40}>
+            {description}
+          </StyledDescription>
+        )}
+        {displayButtons && (
+          <StyledButtonGroup>
+            {showPrimaryButton && (
```

Comment:
can we derive showPrimaryButton and showBasic button from the content instead? Just like helpText here

Reply:
@marthayaputra I'm already deriving them from the button content. I just think it's more readable to name a const rather than inline `!!buttonPrimaryContent` for the condition